client/extensions/woocommerce/lib/order-values/test/fixtures/order-with-refunds.js

```diff
@@ -0,0 +1,80 @@
+/** @format */
+
+export default {
+	line_items: [
+		{
+			id: 15,
+			name: 'Scarf - Striped',
+			product_id: 49,
+			taxes: [
+				{
+					id: 4,
+					total: '5.3964',
+					subtotal: '6.3487',
+				},
+			],
+			price: 42.4915,
+		},
+		{
+			id: 19,
+			name: 'T-Shirt',
+			product_id: 62,
+			taxes: [
+				{
+					id: 4,
+					total: '1.1424',
+					subtotal: '1.1424',
+				},
+			],
+			price: 17.99,
+		},
+	],
+	tax_lines: [
+		{
+			id: 17,
+			rate_code: 'US-CT-CT TAX-1',
+			rate_id: 4,
+			label: 'CT Tax',
+			compound: false,
+			tax_total: '6.54',
+			shipping_tax_total: '0.64',
+		},
+	],
+	shipping_lines: [
+		{
+			id: 16,
+			method_title: 'USPS',
+			method_id: 'usps:3:0',
+			total: '10.00',
+			total_tax: '0.64',
+			taxes: [
+				{
+					id: 4,
+					total: '0.635',
```

Comment:
Will having 0.635 here but 0.64 above (and above in tax_lines too) potentially cause problems/confusion down the road? Could we use the same value in all three places?

Reply:
Again, this is stripped data from a "real" (test) order. 0.635 is the full tax amount, but 0.64 is the display value, since you can't have half a cent. (and that's why we need these tax+etc calculating functions, we can't rely on the display values being totally accurate)